app.cr

```diff
@@ -9,13 +9,20 @@ def headers
   headers
 end
 
-def crystal_repos
+def crystal_repos(sort)
   client = HTTP::Client.new("api.github.com", 443, true)
-  response = client.get("/search/repositories?q=language:crystal&per_page=100", headers)
+  response = client.get("/search/repositories?q=language:crystal&per_page=100&sort=#{sort}", headers)
```

Comment:
I'd love the pull request, because I don't have a lot of time right now. But I agree that it can go on a different patch.